playbook/workshops/contrib/ajar/README.md

```diff
@@ -234,4 +234,174 @@ You've finished Part 1!
 
 ![](img/celebrate.gif)
 
-# More coming soon...
+
+## Part 2
+
+Welcome to part 2 of the Ajar.io workshop!
+
+To recap, in [part 1](#part-1) you learned about:
+
+- **JavaScript variables**: used to store data values. Declared using the `var`
```

Comment:
@paked good suggestion with the variable usage, I'll fix it.

I don't see the advantage of using a table. Maybe you can clarify?